pkg/service/test/test.go

```
@@ -274,7 +275,13 @@ func (t *tester) Test(path string, testReportPath string, generateTestReport boo
 		testSuiteNames = append(testSuiteNames, testSuiteName)
 	}
 
-	sort.Strings(testSuiteNames)
+	sort.SliceStable(testSuiteNames, func(i, j int) bool {
+
+        num1, _ := strconv.Atoi(strings.Split(testSuiteNames[i], "-")[2])
+        num2, _ := strconv.Atoi(strings.Split(testSuiteNames[j], "-")[2])
+
```

is it wise to ignore the errors here?